MediaRecorder.bs

```diff
@@ -245,15 +249,24 @@ interface MediaRecorder : EventTarget {
     the value might be surpassed, not achieved, or only be achieved over a long
     period of time.</li>
 
-    <li>Let |videoKeyFrameInterval| be the value of |recorder|'s internal
-    [=[[VideoKeyFrameInterval]]=] slot if defined, and configure the video encoder to
-    target key frame generation with the interval, meaning the User Agent SHOULD
-    encode a key frame on the next frame after this interval has passed since the
-    last key frame. If |videoKeyFrameInterval| isn't defined, the User Agent will
-    emit key frames as it deems fit.
-    |videoKeyFrameInterval| is a hint for the video encoder and the actual
-    intervals in the recording might be surpassed or not achieved. Note also that
-    encoders will make independent decisions about when to emit key frames.</li>
+    <li>Let |videoKeyFrameIntervalDuration| be the value of |recorder|'s internal
+    [=[[VideoKeyFrameIntervalDuration]]=] slot if defined. Additionally let
+    |videoKeyFrameIntervalCount| be the value of |recorder|'s internal
+    [=[[VideoKeyFrameIntervalCount]]=] slot if defined.
+    If neither of the slots are defined, the User Agent may emit key frames as
+    it deems fit.
+    If only |videoKeyFrameIntervalDuration| is defined, the video encoder
+    SHOULD produce a keyframe on the first frame arriving after
+    |videoKeyFrameIntervalDuration| elapsed since the last key frame.
+    If only |videoKeyFrameIntervalCount| is defined, the video encoder
+    SHOULD produce a keyframe on the first frame arriving after
+    |videoKeyFrameIntervalCount| frames passed since the last key frame.
+    If both are defined, the video encoder SHOULD produce a keyframe on the
+    first frame arriving after both |videoKeyFrameIntervalDuration| elapsed AND
+    |videoKeyFrameIntervalCount| frames passed since the last key frame.
```

Comment:
The text seems to say that if duration is 1 second and count is 10, then a keyframe should be produced after 1 second if the frame rate is > 10, and should be produced after 10 frames if the frame rate is < 10 - "latest one wins".

That seems contrary to the Principle of Least Surprise - should "both" be "either"?

Reply:
Resolving convo, pls reopen if there are still issues.